website/src/styles.css

```diff
@@ -72,3 +72,7 @@ html[data-theme='dark'] .docusaurus-highlight-code-line {
 #view-main > div > div.maplibregl-control-container.mapboxgl-control-container > div.maplibregl-ctrl-bottom-right.mapboxgl-ctrl-bottom-right > details > summary {
   display: none !important;
 }
+
+#docusaurus_skipToContent_fallback > section > div.styled__Container-sc-exnbu5-1.styled__BannerContainer-sc-exnbu5-2.gMcHAv.fYMkHs > h1 {
```

Comment:
That is quite the incantation... We should probably add short comments to these to explain what is going on.

A lot of those class names look auto generated. Are we sure they will not be generated into something else in the future? Can we add a class name in the react code so this is less random looking / more stable / easier to understand?